recipe/meta.yaml

```diff
@@ -0,0 +1,191 @@
+{% set ignore_modules = ignore_modules + " --ignore=test/torchaudio_unittest/models" %}  # [(linux and s390x) or win]
+# these tests import torchaudio.prototype which does not exist in main module
+{% set ignore_modules = ignore_modules + " --ignore=test/integration_tests/prototype --ignore=test/integration_tests/rnnt_pipeline_test.py" %}
+# test to skip because of missing flashlight-text
+{% set ignore_modules = ignore_modules + " --ignore=test/integration_tests/ctc_decoder_integration_test.py" %}
+# test to skip because of missing dependency deep-phonemizer
+{% set ignore_modules = ignore_modules + " --ignore=test/integration_tests/tacotron2_pipeline_test.py" %}
+# Tests to skip because of frame_offset not being accepted as argument of load() function. All load function implementation
+# seem to have a frame_offset parameter except for torchaudio/backend/_no_backend.py::load(), which should not be used since
+# ffmpeg is chosen as backend. More investigation would be necessary but this doesn't seem to be a problem with all other
+# occurrences of the load() function
+{% set ignore_modules = ignore_modules + " --ignore=test/torchaudio_unittest/datasets/tedlium_test.py" %}
+# skipping this test because of URL errors on osx (these are passing locally though)
+{% set ignore_modules = ignore_modules + " --ignore=test/integration_tests/loudness_compliance_test.py" %}  # [osx]
+# stream writer tests fail on windows because another process locks the writing on the output directory (common problem on win)
+{% set ignore_modules = ignore_modules + " --ignore=test/torchaudio_unittest/io/stream_writer_test.py" %}  # [win]
+
+# We don't provide kaldi-io in our distro, so skip tests that test functionality that requires it
+{% set tests_to_skip = "kaldi" %}
+# skip test fail in EffectorTest.test_formats_5_mp3 with "RuntimeError: Failed to open codec: (Invalid argument)"
+# because of missing mp3 codec support on ffmpeg for win-64
+{% set tests_to_skip = tests_to_skip + " or test_formats_5_mp3" %}  # [win]
+
+test:
+  source_files:
+  {% if run_all_unittests %}
+    - test/torchaudio_unittest
+  {% else %}
+    # choosing a subset of tests to prevent resource exhaustion and still run a reasonable amount of unit tests
+    - test/torchaudio_unittest/assets
+    - test/torchaudio_unittest/backend
+    - test/torchaudio_unittest/common_utils
+    - test/torchaudio_unittest/compliance
+    - test/torchaudio_unittest/datasets/__init__.py
+    - test/torchaudio_unittest/datasets/tedlium_test.py
+    - test/torchaudio_unittest/example
+    - test/torchaudio_unittest/functional/__init__.py
+    - test/torchaudio_unittest/functional/functional_cpu_test.py
+    - test/torchaudio_unittest/functional/functional_impl.py
+    - test/torchaudio_unittest/functional/autograd_impl.py
+    - test/torchaudio_unittest/functional/torchscript_consistency_impl.py
+    - test/torchaudio_unittest/io
+    - test/torchaudio_unittest/models/conformer
```

Comment:
I think you can skip the models tests, they're resource-hungry, even if they do run ok it'll save time during builds to remove them.

Reply:
agreed. I was tempted to exclude them before.